bash_completion

```diff
@@ -92,5 +92,5 @@ if [[ -n ${ZSH_VERSION-} ]]; then
   autoload -U +X bashcompinit && bashcompinit
 fi
 
-complete -o default -o nospace -F __nvm nvm
+complete -o default -F __nvm nvm
```

Comment:
Hmm - I feel like that might be safer, in that you can't accidentally tab through more than one argument.